pkg/api/kubernetesagentpool/v20170727/convert.go

```diff
@@ -0,0 +1,23 @@
+package v20170727
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Azure/acs-engine/pkg/agentPoolOnlyApi/v20170831"
+)
+
+
+// Convertv20170831ToAgentPool will convert a Convertv20170831 object into AgentPool=
+func Convertv20170831ToAgentPool(hostedMaster *v20170831.HostedMaster) (*AgentPool, error) {
```

Comment:
Do we want to have two packages (`pkg/api/kubernetesagentpool` and `pkg/api/kubernetesagentpool/v20170727`) with this same exportable function?